src/android/aidl/types/parcelables.md

```diff
@@ -0,0 +1,33 @@
+# Parcelables
+
+Binder for Rust supports sending parcelables directly:
+
+**IBirthdayService.aidl**:
+
+```java
+parcelable BirthdayInfo {
+    String name;
+    int years;
+}
+
+interface IBirthdayService {
+    String wishHappyBirthday(BirthdayInfo info);
+}
+```
+
+**client/src/main.rs**:
+
+```rust,ignore
+let msg = service.wishHappyBirthday(BirthdayInfo {
```

Comment:
I don't remember off the top of my head: is the argument by value or by reference?

Reply:
By reference, since it's an `in` parameter. The code example has been updated.